Content.Server/Fluids/EntitySystems/SmokeOnTriggerComponent.cs

```diff
@@ -0,0 +1,20 @@
+using Content.Shared.Chemistry.Components;
+using Robust.Shared.Audio;
+
+namespace Content.Server.Fluids.EntitySystems;
+
+[RegisterComponent]
+[Access(typeof(SmokeSystem))]
+public sealed class SmokeOnTriggerComponent : Component
```

Comment:
Why is this needed? Can't you use `ChemicalPayloadComponent` with smoke chems?

Reply:
I'm not sure that it is possible to implement this through ChemicalPayload. In addition, the grenade should not be able to replace what is inside. If you want me to use ChemicalPayload, you must provide a couple of examples of entities in which a filled chemical charge is already installed, I did not find such in the game code.

Reply:
pretty sure you are able to:
1. use chemical payload
2. use solutioncontainermanager to initialize it with a smoke mix

since the crowbaring bit is on modular grenade's main component if you dont have that you wont have it do anything

Reply:
I can not do it

And in PayloadCaseComponent there is no way to specify what will be inside.
![image](https://github.com/space-wizards/space-station-14/assets/60344369/ec806d87-bf16-46b5-b8bf-bb59e2dbb3f5)

Reply:
`ChemicalPayloadComponent` uses item slots so you'd have probably to create prototypes for beakers filled with the chems. Or change `ChemicalPayloadComponent` to just use solution directly.